apps/reference/docs/guides/realtime/limits.mdx

```diff
@@ -0,0 +1,66 @@
+---
+id: limits
+title: Realtime Limits
+description: Realtime Rate Limiting
+---
+
+Since deploying a global multi-tenant cluster for Supabase Realtime, we've implimented some rate limits to protect our infrastructure.
+
+Rate limits are mostly configurable per project and our cluster can support millions of concurrent connections if needed. If you're running into any rate limiting and it's causing issues, please [contact support](https://app.supabase.com/support/new).
```

Comment:
I would take out the following few words:

```suggestion
Rate limits are configurable per project and our cluster can support millions of concurrent connections. If you're running into any rate limiting and it's causing issues, please [contact support](https://app.supabase.com/support/new).
```

Reply:
A bit more trimmed from Wenbo's:

> Rate limits are configurable per project and our cluster supports millions of concurrent connections. [Contact support](https://app.supabase.com/support/new) if these limits are causing issues.

No need to say "please" in docs - keep them unemotional and deliver the facts

Reply:
Done